jest.config.js

```diff
@@ -2,5 +2,6 @@
 module.exports = {
   verbose: true,
   preset: "@testing-library/react-native",
+  setupFiles: ["./node_modules/react-native-gesture-handler/jestSetup.js"],
```

Comment:
Will you like to run this within the `jest.setup.ts` instead? 
My worry is that the setup might be all over the place in the future

Reply:
Yep, good point, I've moved it into `jest.setup.ts`